tidb-cloud/serverless-limitations.md

```diff
@@ -48,6 +48,10 @@ We are constantly filling in the feature gaps between TiDB Serverless and TiDB D
 - [Changefeed](/tidb-cloud/changefeed-overview.md) is not supported for TiDB Serverless currently.
 - [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) is not supported for TiDB Serverless currently.
 
+### TTL
+
+- Modifying table's `TTL_JOB_INTERVAL` is not supported for TiDB Serverless currently, `TTL_JOB_INTERVAL` is fixed at `15m` currently.
```

Comment:
```suggestion
### Time to live (TTL)

- In TiDB Serverless, the [`TTL_JOB_INTERVAL`](/time-to-live.md#ttl-job) attribute for a table is fixed at `15m` and cannot be modified. This means that TiDB Serverless schedules a background job every 15 minutes to clean up expired data.
```

Please add the following note to line 130 of `time-to-live.md`:
https://github.com/pingcap/docs/blob/ea1d3c69a831693ab943b38523bfc06c7bd6f731/time-to-live.md?plain=1#L129-L132

```markdown
> **Note:**
> 
> In TiDB Serverless, the `TTL_JOB_INTERVAL` attribute for a table is fixed at `15m` and cannot be modified. This means that TiDB Serverless schedules a background job every 15 minutes to clean up expired data.
```